File: benchmark/corpus/08_memory/e_map_growth.js
// Adds a fresh key to a map on every turn and never removes one.
const seen = new Map();
let i = 0;
while (true) {
  seen.set("key-" + i + "-" + Array.from({ length: 20 }, (_, k) => String(i * k % 97)).join(" "), i); // DANGER
  i++;
}
console.log(seen.size);

File: benchmark/corpus/08_memory/e_map_growth.py
# Adds a fresh key to a map on every turn and never removes one.
seen = {}
i = 0
while True:
    seen["key-%d-%s" % (i, " ".join(str(i * k % 97) for k in range(20)))] = i  # DANGER
    i += 1
print(len(seen))
